ocs_ci/ocs/ui/page_objects/block_and_file.py

```diff
@@ -181,3 +183,53 @@ def get_raw_capacity_card_values(self):
         )
 
         return used, available
+
+    def get_estimated_days_from_consumption_trend(self):
+        """
+        This will get the web element xpaths from views.py and try to get text from those elements
```

Comment:
please remove this autogenerated line
`This will get the web element xpaths from views.py and try to get text from those elements` 
- it is applicable to all classes and functions in UI part of framework